### PR TITLE
Update Neutral Colors as per DSM (Issue #323)

### PR DESCRIPTION
**Pull request by Ashygoyal**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [guidelines](https://github.com/mobi/goponents/blob/master/CONTRIBUTING.md)
- [ ] Tests for the changes have been added
- [ ] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Style Update (CSS)
- [ ] Other... Please describe:

## What is the current behavior?
At several places in the UI, colors were not up to date as per the latest style guide.

Resolves [323](https://github.com/mobi/goponents/issues/323)

## What is the new behavior?
Neutral colors have been updated as per details mentioned in issue description and comments.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/go-lib/src/lib/components/go-search/go-search.component.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
   @include transition(all);
 
   background: $theme-light-bg;
-  border: 1px solid $theme-light-border;
```
Comment (grahamhency) [inline projects/go-lib/src/lib/components/go-search/go-search.component.scss:17]: Are we sure about this one? This change makes the border on the search _very_ light. 

Screen Shot 2020-10-29 at 4 46 35 PM

@kpmurphh can you confirm this for us?

**Review by grahamhency — COMMENTED** [review]
@Ashygoyal just had one question for @kpmurphh in regards to the global search border color, but the rest looks good.

File: projects/go-lib/src/lib/components/go-search/go-search.component.scss (hunk 1 above)
Comment (kpmurphh) [reply projects/go-lib/src/lib/components/go-search/go-search.component.scss:17]: @grahamhency @Ashygoyal I can't tell what color it is from this code or this image, but it should match the other inputs border color, which is #D3D3D3.

Comment (Ashygoyal) [reply projects/go-lib/src/lib/components/go-search/go-search.component.scss:17]: @kpmurphh As per the DSM [guidelines](https://tangoe.invisionapp.com/dsm/tangoe/tangoe-ds/folder/colors/5c9435edae717d86ac2b0c6d), the search border color should be `#F0F0F0`
image

I hope I am referring to the updated DSM URL. Can you please confirm.
For this PR, I'll make the search border as `#D3D3D3`.

Comment (Ashygoyal) [reply projects/go-lib/src/lib/components/go-search/go-search.component.scss:17]: @grahamhency I have updated this PR, ready for one last review!
Thanks!

**Review by grahamhency — APPROVED** [review]
Looks good, let's :shipit:

**Pull request merged**